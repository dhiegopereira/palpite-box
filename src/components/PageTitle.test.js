import React from 'react';
import renderer from 'react-test-renderer';
import {render, screen} from '@testing-library/react'
import PageTitle from './PageTitle'


describe('PageTitle Component', () => {
    it('Return title', () => {
        const tree = renderer
            .create(<PageTitle title="Diego"></PageTitle>)
            .toJSON();
        expect(tree).toMatchSnapshot();        
    })
})