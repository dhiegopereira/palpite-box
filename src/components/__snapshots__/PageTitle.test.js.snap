// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Component Return title 1`] = `null`;
